MAUI/ImageEditor/getting-started.md

```diff
@@ -0,0 +1,131 @@
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Getting Started with .NET MAUI ImageEditor (SfImageEditor)
+
+
+
+
+
+## Creating an application using the .NET MAUI ImageEditor
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfImageEditor to your project, open the NuGet package manager in Visual Studio, search for `[Syncfusion.Maui.ImageEditor]` and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.ImageEditor` in XAML or C# code.
+
+4. Initialize `SfImageEditor.`
+
+{% tabs %}
+{% highlight XAML hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:imageEditor="clr-namespace:Syncfusion.Maui.ImageEditor;assembly=Syncfusion.Maui.ImageEditor">
+
+    <imageEditor:SfImageEditor Source="image_name.extension" />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.ImageEditor;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfImageEditor imageEditor = new SfImageEditor();
+        this.Content = imageEditor;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+##Loading an image to image editor
+
+Refer to the following steps to add an image to the pcl project:
+
+1. Locate the "Resources" folder in your .NET MAUI project. This folder is typically located in the project's root directory.
+2. Right-click on the "Resources" folder in the project structure view or Solution Explorer.
+3. From the context menu, select "Add" and then "Existing Item." This will open a file selection dialog.
+4. Browse to the location on your computer where the image file is stored.
+5. Select the image file you want to add to the "Resources" folder.
+6. Click the "Add" button to add the image file to the project.
+7. After the image has been imported, ensure whether the image Build Action has been set to `EmbeddedResource`.

```

Comment:
Remove point 7, no need for this

7. After the image has been imported, ensure whether the image Build Action has been set to `EmbeddedResource`.

Reply:
removed